AY2021S2-CS2103-T14-1/tp
# Docs(DG): Update Logic component section

## What this does
Closes #356 by updating the Logic component section, its links and its diagrams.
Fixes #273 by updating `LogicManager` in the Logic component UML diagram to point to storage.

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the DG's Logic component section renders correctly and makes sense.
